cluster/saltbase/salt/calico/master.sls

```diff
@@ -0,0 +1,35 @@
+{% if pillar.get('policy_provider', '').lower() == 'calico' %}
+
+calico-etcd:
+  cmd.run:
+    - unless: docker ps | grep calico-etcd
+    - name: >
```

Comment:
Yeah, it may be that we want to re-open those discussions.  I think it's something we'd be interested in doing, but likely requires some non-trivial changes, so if we go down that path we'd want to start with something like this and iterate.

I agree though - for now, this etcd should be in the same pod as the policy controller.